seong-geun/app.js

```diff
@@ -115,22 +129,55 @@ app.get("/u_posts", async (req, res) => {
    `,
     [userId]
   );
+
   res.status(200).json({ data: userspost });
-  const { id, profile_image, username, age, email } = req.body;
+});
 
+// 게시글 수정하기
+app.patch("/m_posts", async (req, res) => {
```

Comment:
```suggestion
app.patch("/posts", async (req, res) => {
```
엔드포인트(목적지로 비유를 들 수 있습니다.)를 설계, 작성 할 때에는 언더바(`_`)를 사용하는 것을 지양해야 합니다. 또한 `m_`이라고 작성해주신 앞부분을 제외하더라도 `/posts`라는 엔드포인트와, 수정을 의미하는 HTTP method인 `patch`의 조합만으로 what과 how 두가지를 모두 나타낼 수 있습니다. "`posts`를 `patch` 하겠다" 처럼요. 

자세한 내용은 금주에 있는 RESTful API 세션에서 다룰 예정이니 지금은 간단히 인지만 하고 넘어가시면 될 것 같습니다. 수정해주세요~